resinhup.py

```diff
@@ -117,6 +117,13 @@ def main():
         log.error("Resinhup is not supported for the resinOS version requested.")
```

Comment:
I would say that beta version shouldn't be updated. If you are running a beta version you are aware that resinhup will not work for you so you will have to reflash at some point. Now, in order to detect this we can do the following:
1. if beta bail out
2. if force, update even beta.

I'm a little concerned about the force flag too. The peoblem is that if you force the version it will automatically force the fingerprints check too. Maybe we should use additional flags? Or some other smarter mechanism.